include/Chess.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include <iostream>
+#include <Windows.h>
+#include <string>
+
+using std::cout;
+using std::cin; 
+using std::endl;
+using std::string; 
+
+const int _SIZE = 21;
+
+class Chess {
+	unsigned char m_board[_SIZE][_SIZE] = { 0 };
+	bool m_turn = true;
+	string m_boardString;
+	string m_input;
+	string m_msg = "\n";
+	string m_errorMsg = "\n";
+	int m_codeResponse;
+
+	void clear() const;
+	void setFrames();
+	void setPieces();
+	void show() const;
+	void displayBoard() const;
+	void showAskInput() const;
+	bool isSame() const;
+	bool isValid() const;
+	bool isExit() const;
+	void excute();
+	void doTurn();
```

Comment:
Pay attention, is there a correct division between the UI and the logic? Try to keep the class responsible for the logic from doing anything relatead to UI, and vice versa